pkg/proxy/iptables/proxier_test.go

```diff
@@ -281,7 +281,7 @@ func TestDeleteEndpointConnections(t *testing.T) {
 	// Run the test cases
 	for _, tc := range testCases {
 		priorExecs := fexec.CommandCalls
-		priorGlogErrs := klog.Stats.Error.Lines()
+		priorLogErrs := klog.Stats.Error.Lines()
```

Comment:
shouldn't this be priorKlogErrs for consistency?

Reply:
I was following what we changed here - https://github.com/kubernetes/kubernetes/pull/71936#pullrequestreview-184750363 if we ever change the logger under the hood it might be easier for us to manage this later if it's less implementation specific. Thoughts?

Reply:
The way I was thinking about it, the klog layer was itself the abstraction layer which would allow us to change the implementation without changing the interface.

Reply:
@dims have any input here? can always leave change iptables to  `priorKlogErrs` and leave `staging/src/k8s.io/client-go/rest/request.go` without implementation reference…

Reply:
I'm in agreement, I feel like tying the implementation with the function name will put us in a place like we are with https://github.com/kubernetes/kubernetes/pull/72023#pullrequestreview-184976864 where it's an exported function which we don't technically know who is referencing it besides what is in `kubernetes/kubernetes`